sql/core/src/test/resources/sql-tests/inputs/ansi/conditional-functions.sql

```diff
@@ -1,6 +1,41 @@
 -- Tests for conditional functions
-CREATE TABLE t USING PARQUET AS SELECT c1, c2 FROM VALUES(1, 0),(2, 1) AS t(c1, c2);
+--SET spark.sql.maxMetadataStringLength = 500
```

Comment:
what does this do?

Reply:
Explain include the location metadata which is a long string. Default length will cause `location...` which failed to comprare plan.